adapt/engine.py

```diff
@@ -368,3 +434,43 @@ def register_intent_parser(self, intent_parser, domain=0):
             self.register_domain(domain=domain)
         self.domains[domain].register_intent_parser(
             intent_parser=intent_parser)
+
+    def drop_intent_parser(self, parser_names, domain):
+        """Drop a registered intent parser.
+
+        Arguments:
+            parser_names (list, str): parser names to drop.
+            domain (str): domain to drop from
+
+        Returns:
+            (bool) True if an intent parser was dropped else false.
+        """
+        return self.domains[domain].drop_intent_parser(parser_names)
+
+    def drop_entity(self, domain, *, entity_type=None, match_func=None):
+        """Drop all entities mathching the given entity type or match function.
+
+        Arguments:
+            domain (str): intent domain
+            entity_type (str): entity name to match against
+            match_func (callable): match function to find entities
+
+        Returns:
+            (bool) True if vocab was found and removed otherwise False.
+        """
+        return self.domains[domain].drop_entity(entity_type=entity_type,
+                                                match_func=match_func)
+
+    def drop_regex_entity(self, domain, *, entity_type=None, match_func=None):
```

Comment:
did you mean to leave the `*` args in place here?

Reply:
cc @forslund super close, I think this is the last thing!

Reply:
I did mean to leave them there to require keyword arguments for these. But I'm not married to the idea so I'll just remove them.

Reply:
There, keyword only spec has been removed.